VK/VK/Cells/PhotoCollectionViewCell.swift

```diff
@@ -22,7 +22,7 @@ final class PhotoCollectionViewCell: UICollectionViewCell {
 
     // MARK: - Public Methods
 
-    func configure(_ photoNames: [String]) {
-        friendImageView.loadImage(photoNames.first ?? Constants.emptyString, networkService)
+    func configure(_ indexPath: IndexPath, _ photoName: String, _ photoCacheService: PhotoCacheService) {
```

Comment:
```suggestion
    func configure(photoName: String, photoCacheService: PhotoCacheService) {
```
_ не ставь тут с наружи должно быть видно что передаем